drivers/bluetooth/hci/Kconfig

```
@@ -40,6 +40,24 @@ config BT_HCI_IPC
 	  Bluetooth HCI driver for communication with another CPU
 	  using the IPC subsystem.
 
+config BT_HCI_IPC_SEND_RETRY_COUNT
+	int "HCI IPC send retry count"
+	depends on BT_HCI_IPC
+	default 3
+	help
+	  Number of times to retry sending a message via IPC.
+	  The retries occur in case of a failure to send a message
+	  through IPC (-ENOMEM returned by ipc_service_send).
+
+config BT_HCI_IPC_SEND_RETRY_DELAY_US
+	int "HCI IPC send retry delay (us)"
+	depends on BT_HCI_IPC
+	default 75
+	help
+	  Delay in microseconds between retries when sending a message via IPC.
+	  When a single tick (CONFIG_SYS_CLOCK_TICKS_PER_SEC) is bigger then
+	  the retry delay the k_busy_wait function is used. Set with care.
```

```suggestion
config BT_HCI_IPC_SEND_RETRY_COUNT
	int "HCI IPC send retry count [EXPERIMENTAL]"
	depends on BT_HCI_IPC
	default 3
	select EXPERIMENTAL
	help
	  Number of times to retry sending a message via IPC.
	  The retries occur in case of a failure to send a message
	  through IPC (-ENOMEM returned by ipc_service_send).

config BT_HCI_IPC_SEND_RETRY_DELAY_US
	int "HCI IPC send retry delay (us) [EXPERIMENTAL]"
	depends on BT_HCI_IPC
	default 75
	select EXPERIMENTAL
	help
	  Delay in microseconds between retries when sending a message via IPC.
	  When a single tick (CONFIG_SYS_CLOCK_TICKS_PER_SEC) is bigger then
	  the retry delay the k_busy_wait function is used. Set with care.
```

Suggest to add experimental values so they are not considered stable Kconfig options just yet (in case we want to remove them later)

---

I would rather not use experimental since it will be used by default in the HCI over IPC at all times. This will add experimental label to every build, which is not desired.

---

Yeah that makes sense. Somehow I had expected the retries not to be enabled by default as there hasn't been any Zephyr issue raised on this, so thought this was for some uncommon case. 

I'll leave this up to the maintainers @alwa-nordic @jhedberg to decide whether it should be EXPERIMENTAL and/or enabled by default